var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -15,9 +15,17 @@ class Relion(CMakePackage, CudaPackage):
     homepage = "http://http://www2.mrc-lmb.cam.ac.uk/relion"
     git      = "https://github.com/3dem/relion.git"
 
-    version('3.0.7', tag='3.0.7')
```

Comment:
We don't normally remove old releases so that people can reproduce their results later, even if those results may contain bugs.

Reply:
Thanks, will add back for this purpose.